sycl/include/CL/sycl/ONEAPI/sub_group.hpp

```diff
@@ -745,5 +754,16 @@ inline sub_group this_sub_group() {
 }
 
 } // namespace ONEAPI
+
+namespace experimental {
+inline ONEAPI::sub_group this_sub_group() {
```

Comment:
can we use sycl::sub_group here as soon as it has been recently changed: https://github.com/intel/llvm/blob/sycl/sycl/include/CL/sycl/sub_group.hpp?
Less deprecation warning is better...